src/main/java/thaumicenergistics/common/inventory/HandlerItemEssentiaCell.java

```diff
@@ -605,47 +607,49 @@ public List<IAspectStack> getStoredEssentia() {
      *
      * @return
      */
+    @Override
     public long getTotalBytes() {
         return this.totalBytes;
     }
 
-    /**
-     * Total number of types the cell can hold.
-     *
-     * @return
-     */
-    public int getTotalTypes() {
-        return this.totalTypes;
-    }
+    // /**
+    // * Total number of types the cell can hold.
+    // *
+    // * @return
+    // */
+    // public int getTotalTypes() {
+    // return this.totalTypes;
+    // }
```

Comment:
Why are these just commented out?

Reply:
I make a new interface `ICellCacheRegistry` in AE2, that helps me to collect cell bytes and type info, and I accidentally use the same name in the interface. But return is long in the interface.

Reply:
Yeah but you should terminate them if that's the case

Reply:
OK, I'm working on it.